docs/kyma/docs/036-inst-custom-component-installation.md

```diff
@@ -0,0 +1,67 @@
+
+To disable a component from the list of components that you install with Kyma, remove this component's entries from the appropriate file. The file differs depending on whether you install Kyma from the release or from sources, and if you install Kyma locally or on a cluster. The version of your component's deployment must match the version that Kyma currently supports.
+
+### Installation from the release
+
+1. Download the [newest version](https://github.com/kyma-project/kyma/releases) of Kyma.
+2. Customize installation by removing a component from the list of components in the Installation resource. For example, to disable the Application Connector installation, remove this entry:
+    ```
+    name: "application-connector"
+    namespace: "kyma-system"
+    ```
+  * from the `kyma-config-local.yaml` file for the **local** installation
+  * from the `kyma-config-cluster.yaml` file for the **cluster** installation
+
+
+3. Follow the installation steps described in the [Install Kyma locally from the release](#installation-install-kyma-locally-from-the-release) document, or [Install Kyma on a GKE cluster](#installation-install-kyma-on-a-gke-cluster) accordingly.
+
+### Installation from sources
+
+1. Customize installation by removing a component from the list of components in the following Installation resource:
+  * [installer-cr.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr.yaml.tpl) for the **local** installation
+  *  [installer-cr-cluster.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster.yaml.tpl) for the **cluster** installation
+
+2. Follow the installation steps described in the [Install Kyma locally from sources](#installation-install-kyma-locally-from-sources) document, or [Install Kyma on a GKE cluster](#installation-install-kyma-on-a-gke-cluster) accordingly.
+
+### Verify the installation
+
+1. Check if all Pods are running in the `kyma-system` Namespace:
+  ```
+  kubectl get pods -n kyma-system
+  ```
+2. Sign in to the Kyma Console using the `admin@kyma.cx` email address as described in the [Install Kyma locally from the release](#installation-install-kyma-locally-from-the-release) document.
+
+
+## Add a component
+
+>**NOTE:** This section assumes that you already have your Kyma Lite local version installed successfully.
+
+To install a component that does is not installed with Kyma by default, run an appropriate `helm install` command inside the `resources` directory:
```

Comment:
we can simplify this section, remove all those custom helm installations and simply give instruction which I just tested.

If you want to install component that is not available in kyma lite:
1. Edit **Installation** resource: `kubectl edit installation kyma-installation`
2. Uncomment the component you want to install in the list of components and save your change
3. Trigger installation `kubectl label installation/kyma-installation action=install`
4. Monitor installation progress with `./scripts/is-installed.sh` triggered from `installation` directory